lib/projectile.js

```diff
@@ -31,28 +31,17 @@ Projectile.prototype.movement = function() {
 };
 
 function powerSetting(power) {
-  if(power <= 20) {
-    return 0.5;
-  } else if (20 < power && power <= 40) {
-    return 0.6;
-  } else if (40 < power && power <= 60) {
-    return 0.7;
-  } else if (60 < power && power <= 80) {
-    return 0.8;
-  } else if (80 < power && power <= 100) {
-    return 0.9;
-  } else if (100 < power && power <= 120) {
-    return 1.1;
-  } else if (120 < power && power <= 140) {
-    return 1.2;
-  } else if (140 < power && power <= 160) {
-    return 1.35;
-  } else if (160 < power && power <= 180) {
-    return 1.45;
-  } else if (180 < power && power <= 200) {
-    return 1.6;
+  var min = 0;
+  var max = 20;
+  var result = 0.5;
```

Comment:
Does this particular value come from something variable? I'm just a little unclear about why "result" isn't the 'result' of some other equation. Should it always be 0.5?